testhelper/helpers.go

```diff
@@ -1,5 +1,26 @@
 package testhelper
 
-import "github.com/mia-platform/go-crud-service-client/internal/types"
+import (
+	"encoding/json"
+
+	"github.com/mia-platform/go-crud-service-client/internal/types"
+)
 
 type Filter types.Filter
+
+func ParseResponseToNdjson[TResource any](response []TResource) (string, error) {
```

Comment:
Is it possible to import this function in other packages?

Reply:
yes it can be imported by third party packages as `testhelper.ParseResponseToNdjson(...`